test/unit_tests/graphql/test_subscriptions.py

```diff
@@ -877,6 +877,7 @@ def test_single_subscription(test_client, product_type_1_subscriptions_factory,
         "totalItems": 1,
     }
     assert subscriptions[0]["subscriptionId"] == subscription_id
```

Comment:
subscription = subscriptions[0]

and then using that is a bit easier on the eye.

Reply:
its done on many places and we have a story for changing tests to assert with snapshots. so don't find it usefull to update here.